src/test/java/FizzBuzzTest.java

```diff
@@ -0,0 +1,44 @@
+import com.tcs.java8.utility.FizzBuzzUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.util.StringUtils;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class FizzBuzzTest {
+
+    @Test
+    public void shouldReturnFizzWhenInputNumberDividedBy3() {
+        Assert.assertEquals("fizz", FizzBuzzUtils.getFizzBuzzResponse(9));
+    }
+
+    @Test
+    public void shouldReturnBuzzWhenInputNumberDividedBy5() {
```

Comment:
Method naming is not meeting the exact requirement in the condition it should also meet the number divided by 15 requirement
shouldReturnFizzWhenInputNumnerDivisibleBy5AndNotBy15